Languages/Android/questions.json

```diff
@@ -514,6 +514,15 @@
       "trueOrFalse": false,
       "shortAns": "",
       "tag": "asynctask"
+    },
+    {
+      "id": 57,
+      "question": "How many types of activity lifecycle are there in android?",
```

Comment:
Is this question suppose to be multiple choice or a boolean? 

Multiple choice questions have a shortAns. 
Boolean questions should be asked in a way that can be answered "true" or "false". 

IE: 
There are 8 lifecycle methods of an Android Activity?

Also please add at least 3 different questions.